run-tests.js

```diff
@@ -19,7 +19,10 @@ const tests = spawn(cmd, ['tests'], {
   windowsVerbatimArguments: true,
 });
 
-tests.on('close', () => {
-  console.log(`Tests finished! Closing server http://localhost:${port}`);
+tests.on('close', (code) => {
+  console.log(`Tests finished with exit code ${code}.`);
+  process.exitCode = code;
```

Comment:
The run-tests.js script was exiting with code 0 even when tests failed, meaning CI wasn't detecting it as an error.

Reply:
Yeah, I added it some of the newer scripts. https://github.com/percy/percy-cypress/blob/master/run-tests.js#L21-L27 I want a `@percy/oss` package or something so we can keep track of those files in a single place